src/presentation/pages/EndPage.js

```diff
@@ -21,20 +23,55 @@ function EndPage(){
     </motion.div> */}
     
 
-    <motion.div className="End-right-content"
-     initial={{ y: 0, x: 0 }}
-     animate={{ y:0, x: 0}}
-     transition={{ duration: 1, ease: animationConstants.easing }}
-     variants={animationConstants.stagger}
+    <motion.div className="End-left-content"
+    initial={{x: 0}}
+    animate={{x: "-50%"}}
+    transition={animationConstants.transitionEnd}
     >
       <motion.svg xmlns="http://www.w3.org/2000/svg" fill="none" viewBox="0 0 24 24" strokeWidth="1.5" stroke="green" initial="hidden"
       animate="visible" className="w-6 h-6" width="50vw" height="50vw" data-testid="svg-container">
       <motion.path strokeLinecap="round" strokeLinejoin="round" variants={drawSuccess} d="M9 12.75L11.25 15 15 9.75M21 12a9 9 0 11-18 0 9 9 0 0118 0z" />
       </motion.svg>
     </motion.div>
+    
+    <motion.div className="End-text"
+      initial={animationConstants.containerDropIn.initial}
+      animate={{opacity: 1, height: "85vh"}}
```

Comment:
You may want to consider extracting some of these into their AnimationConstants.js and importing them here to make this more readable!